app/invocation/invocation.tsx

```diff
@@ -209,16 +213,9 @@ export default class InvocationComponent extends React.Component<Props, State> {
     });
     const isBazelInvocation = this.state.model.isBazelInvocation();
     const fetchBuildLogs = () => {
-      return rpcService.service
-        .getEventLogChunk(
-          new eventlog.GetEventLogChunkRequest({
-            invocationId: this.props.invocationId,
-            minLines: maxInt32, // int32 max value: max number of lines we can request.
-          })
-        )
-        .then((response: eventlog.GetEventLogChunkResponse) => {
-          return new TextDecoder().decode(response.buffer || new Uint8Array());
-        });
+      rpc_service.downloadBytestreamFile("//buildlog", "bytestream://", this.props.invocationId, {
+        attempt: this.state.attempt.toString(),
```

Comment:
For my own understanding - is the `attempt` needed so that the downloaded logs match the current invocation attempt that is visible in the UI?